plugins/org.eclipse.emf.cdo.ecore/src/org/eclipse/emf/cdo/ecore/impl/EModelElementImpl.java

```diff
@@ -50,6 +50,11 @@
  */
 public abstract class EModelElementImpl extends CDOObjectImpl implements EModelElement
 {
+
+  /**
+   * An extensible set of bit flags.
+   */
+  protected int eFlags;
```

Comment:
This might be needed to make compile errors go away somewhere, but this flag is used to represent state, typically of boolean fields so having state that's not properly persisted by CDO is likely a very bad thing.

Reply:
Hello, and thanks a lot for your feedback. I was doubting it was the right solution indeed, and was prepared for remarks...

In UML metamodel, it seems that the flag is also used to store "runtime/lifecycle" info, such as "adapting", or "detached".  (for instance in `org.eclipse.uml2.uml.internal.impl.ElementImpl.eAdapters()` or `org.eclipse.uml2.uml.internal.impl.ElementImpl.eBasicSetContainer(InternalEObject, int, NotificationChain)`

For classical model state info access, the cdo-ification tool is correcly converting eFlags usage into relevant methods, but I understand that for those corner cases, it's better to fix the issue locally rather than reintroducing this flag for everyone.

I'll have to better understand the cdo-migration tool to determine if it can be directly fixed in the cdo-ified version of UML or if it also requires a fix in this latter. 

Thanks again!